dynacred/src/group/groupContract.ts

```diff
@@ -0,0 +1,169 @@
+import { Group } from "@dedis/kyber";
+import { GroupDefinition, IGroupDefinition } from "./groupDefinition";
+
+export const ENCODING: BufferEncoding = "hex";
+
+export interface IGroupContract {
+    id: string;
+    groupDefinition: IGroupDefinition;
+    signoffs: string[];
+    successor: string[];
+}
+
+/**
+ * Class container representing a group contract
```

Comment:
There should be some more information:
- what it's used for
- how to initialize it
- what other classes it interacts